frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -586,6 +617,135 @@ export default function MotionToWaiveFees_Web({ data }) {
           dataKey="person.financialAssets"
           name="financialAssets"
         />
+        {data.person.financialAssets && (
+          <React.Fragment>
+            <CheckBox
+              label="Bank or credit union"
+              dataKey="person.financialBankOrCreditUnion"
+              name="bankOrCredit"
+            />
+            {data.person.financialBankOrCreditUnion && (
+              <TextInputGroup
+                groupLabel="Bank or credit union"
+                inputs={bankOrCredit}
+                buttonLabel="Bank or credit union"
+                maxInputs={2}
+                dataKey="person.bankOrCredit"
+              />
+            )}
+            <CheckBox
+              label="Stocks, bonds, securities, money market account, "
+              dataKey="person.financialStocksSecuritiesMoney"
+              name="stocksSecurities"
+            />
+            {data.person.financialStocksSecuritiesMoney && (
+              <TextInputGroup
+                groupLabel="Stocks and securities"
+                inputs={otherAssets}
+                maxInputs={1}
```

Comment:
What's the purpose of using TextInputGroup with `maxInputs` set to 1? Couldn't you just render the inputs inside of this web form component instead?